IdentityCore/src/cache/MSIDToken.m

```diff
@@ -204,6 +285,34 @@ - (instancetype)initWithTokenResponse:(MSIDTokenResponse *)response
     return self;
 }
 
+#pragma mark - Private
+
+- (NSMutableOrderedSet<NSString *> *)scopesFromString:(NSString *)scopesString
+{
+    if (!scopesString)
+    {
+        return nil;
+    }
+    
+    NSMutableOrderedSet<NSString *> *scope = [NSMutableOrderedSet<NSString *> new];
+    NSArray *parts = [scopesString componentsSeparatedByString:@" "];
+    for (NSString *part in parts)
+    {
+        if (![NSString msidIsStringNilOrBlank:part])
+        {
+            [scope addObject:part.msidTrimmedString.lowercaseString];
+        }
+    }
+    
+    return scope;
+}
+
+- (NSString *)scopesToString:(NSOrderedSet<NSString *> *)scopes
```

Comment:
move this to a helper or category, we will need it in other places as well

Reply:
done